### PR TITLE
improvement: favicon cache reloading improvement as per #12403

### LINKED ISSUE #12403: Favicon cache busting
"To refresh your website’s favicon you can force browsers to download a new version by adding ?v=2 to the filename.

This is especially helpful in production to make sure the users get the new version."

```           
<link rel="icon" href="/favicon.ico?v=2" />           
 ```   
https://markodenic.com/html-tips/ #9

### PR DESCRIPTION
**Pull request by 0xfaustian**
Closes: #12403

### CODE REVIEW

File: feature-libs/organization/administration/components/shared/sub-list/assign-cell.component.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { Observable } from 'rxjs';
 @Component({
   selector: 'cx-org-assign-cell',
   template: `
-    <button *ngIf="hasItem" (click)="toggleAssign()" class="link">
```
Comment (saad-mo) [inline feature-libs/organization/administration/components/shared/sub-list/assign-cell.component.ts]: pls undo this change

**Comment by saad-mo** [comment]
make sure to link the issue to this PR as well. usually, if you add `Closes #XXX` it will automatically link the PR and issue. you could also do this in the right side bar under "Linked issues"

**Review by saad-mo — APPROVED** [review]

**Review by Matejk00 — APPROVED** [review]
